### ISSUE TITLE
NTS Service

### ISSUE THREAD

**Issue by MyHeadInTheClouds** [janbar/noson-app#96]
Thanks for your application - its great.
NTS Radio service displays no streams when you click through. This works on other sonos players. All other music services work as expected.

**Comment by janbar** [comment]
I tried and get the following log:

```
(SONOS)SendRequest: GET /pmap2.xml HTTP/1.1
Host: www.nts.live:80
User-Agent: Linux UPnP/1.0 Sonos/36.4-41270 (ACR_noson)
Connection: close
Accept-Charset: utf-8
Accept-Encoding: gzip, deflate


(SONOS)GetResponse: HTTP/1.1 301 Moved Permanently
(SONOS)GetResponse: Server: CloudFront
(SONOS)GetResponse: Date: Mon, 25 Nov 2019 21:33:17 GMT
(SONOS)GetResponse: Content-Type: text/html
(SONOS)GetResponse: Content-Length: 183
(SONOS)GetResponse: Connection: close
(SONOS)GetResponse: Location: https://www.nts.live/pmap2.xml
(SONOS)GetResponse: X-Cache: Redirect from cloudfront
(SONOS)GetResponse: Via: 1.1 87714f2c4c5d69d88a49430daf7bbfbc.cloudfront.net (CloudFront)
(SONOS)GetResponse: X-Amz-Cf-Pop: CDG53-C1
(SONOS)GetResponse: X-Amz-Cf-Id: PWHlc4BtJU3CqJagvigGFeHH-GJ7Ggik5pUcBsFp_pVP7ZigSRaNbg==
(SONOS)GetResponse: 
(SONOS)Init: invalid response
```
The cause is the service returns a location for redirection. Until now I disable all redirection for security reason. I will retry enabling redirection and let you kown...

**Comment by janbar** [comment]
Allowing redirection make the service works

<img width="336" alt="Capture d’écran 2019-11-26 à 00 11 40" src="https://user-images.githubusercontent.com/2353635/69585987-887a5c00-0fe1-11ea-891f-23a79c4090eb.png">

I include the fix for the next release.

**Comment by janbar** [comment]
Fixed by the commit 8fa06669787c0153c2e458d291d89d48dd082506

**Comment by janbar** [comment]
Fixed in release 4.0.1
